entrypoint.sh

```diff
@@ -3,7 +3,7 @@
 cd "$GITHUB_WORKSPACE"
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-dotenv-linter ${INPUT_DOTENV_LINTER_FLAGS} \
+dotenv-linter --no-color ${INPUT_DOTENV_LINTER_FLAGS} \
```

Comment:
```suggestion
dotenv-linter --quiet --no-color ${INPUT_DOTENV_LINTER_FLAGS} \
```

Please add the `--quiet` argument too.